external/fv3fit/fv3fit/reservoir/model.py

```diff
@@ -28,8 +44,8 @@ def __init__(
         reservoir: Reservoir,
         readout: ReservoirComputingReadout,
         rank_divider: RankDivider,
+        autoencoder: ReloadableTransfomer,
```

Comment:
All trained models have an autoencoder saved with them, and the `predict` method now assumes that there is an autoencoder, so I made this a required arg.